### ISSUE TITLE
output electron distribution function in image header

### ISSUE THREAD

**Issue by gnwong** [AFD-Illinois/ipole#79]
the eDF (and associated variables) should be saved in image output files. as far as I can tell, they're not currently included. should be an easy fix, once we decide where to put the output functionality.

thanks to Koushik for noticing this problem

**Comment by bprather** [comment]
https://github.com/AFD-Illinois/docs/wiki/Image-Format#electrons-headerelectrons ?

**Comment by gnwong** [comment]
I don't think so. That's set https://github.com/AFD-Illinois/ipole/blob/master/model/iharm/model.c#L892. I'm talking about this parameter https://github.com/AFD-Illinois/ipole/blob/master/src/par.c#L137 along with associated things like https://github.com/AFD-Illinois/ipole/blob/c0f7de63982d494a4863978ad16ff3dc2711220f/src/symphony/params.h#L49

**Comment by bprather** [comment]
Oh, the emission type.  That's a bit more than just the eDF, since you can use it for a bunch of things related to emissivities and not every value is documented.  I'd initially avoided recording it (and the similarly untested powerlaw params) for that reason.

I don't really have time to put in and test the new values, but they should be simple enough to add to that folder in some consistent way when writing the file header, happy to accept that PR quickly.
